pages/whats_new.md

```diff
@@ -7,22 +7,54 @@ breadcrumb:
+This site launched in December 2015 with the full data and contextual information of the 2015 USEITI Report. In response to user feedback in early 2015, we initiated a significant effort to make the site easier to understand.
 
-* **Production data:** Sales volumes and revenue have long been used to estimate how much of each resource was produced. Here, for the first time, you can find more precise data on what resources were produced on federal lands. [Explore federal production data.]({{ site.baseurl }}/explore/#federal-production)
-* **Federal revenue by company:** See what revenue the government received from production on federal lands, sorted by company. [Explore federal revenue by company.]({{ site.baseurl }}/explore/#revenue)
-* **Reconciliation data:** A number of companies were asked to submit data for reconciliation with Department of Interior revenue data. Of the 45 companies invited, 35 participated. [Download reconciliation data.]({{ site.baseurl }}/downloads/reconciliation/)
-* **How it works:** Beyond the numbers, we've included new narratives explaining how the whole system of leasing federal land and collecting royalties on extracted materials works in the U.S. [Learn how it works.]({{ site.baseurl }}/how-it-works/)
+This version of the site includes new content, new state-level datasets, and completely new ways of exploring all data.
 
+## State and national profile pages
+
+In our research, users looked for information about their state and expressed interest in seeing data about their area across multiple datasets. New regional profile pages provide a more complete picture of extractive industries at the national and subnational levels. To see how this works, explore:
+
+* The [national profile page]({{ site.baseurl }}/explore/)
+* State profiles like [Texas]({{ site.baseurl }}/explore/), [Utah]({{ site.baseurl }}/explore/), or [California]({{ site.baseurl }}/explore/)
+* Offshore areas, like the [Gulf of Mexico]({{ site.baseurl }}/explore/offshore-gulf/) or [offshore Alaska areas]({{ site.baseurl }}/explore/offshore-alaska/)
+
+The profile pages also include more granular **disbursements data**, which shows how the Office of Natural Resources Revenue (ONRR) distributes revenue from the extractive industries to [national funds and tribes]({{ site.baseurl }}/explore/#federal-disbursements) and to state governments (for example, [Arizona]({{ site.baseurl }}/explore/AZ/#disbursements) or [Louisiana]({{ site.baseurl }}/explore/LA/#disbursements)).
+
+USEITI also invited states to opt in to the reporting process, with the goal of making state-level data more navigable for the public. In 2016, three states worked with the {{ "Independent Administrator" | term }} to provide **additional state-level data** and contextual information to be integrated in their state profile pages: [Alaska]({{ site.baseurl }}/explore/AK/), [Montana]({{ site.baseurl }}/explore/MT/), and [Wyoming]({{ site.baseurl }}/explore/WY/).
```

Comment:
I think Independent Administrator needs to be `{{ "Independent Administrator" | term: "Independent Administrator (IA)" }}`